src/common/moderate.ts

```diff
@@ -0,0 +1,101 @@
+const badNounsReplacements = [
+    "pasta",
+    "kebab",
+    "cake",
+    "potato",
+    "woman",
+    "computer",
+    "java",
+    "hamburger",
+    "monster truck",
+    "osu!",
+    "Ukrainian ball in search of gas game",
+    "Anime",
+    "Anime girl",
+    "good",
+    "keyboard",
+    "NVIDIA RTX 3090 Graphics Card",
+    "storm",
+    "queen",
+    "single",
+    "umbrella",
+    "mosque",
+    "physics",
+    "bath",
+    "virus",
+    "bathroom",
+    "mom",
+    "owner",
+    "airport",
+    "Avast Antivirus Free",
+    "buckingham palace",
+    "glorp zeep",
+    "lithium mines",
+    "NDA required",
+    "gnarp",
+    "ninachat membership",
+    "device",
+    "nina restaurant",
+    "negative wealth",
+    "koda",
+    "zander",
+    "radiohead",
+    "fumo",
+    "adobe astronaut",
+    "piracy",
+    "ninacord",
+    "devilbro",
+    "32gb Trident Z @3200",
+    "ultrakill"
```

Comment:
style: trailing comma on multiline objects for better diffs
```suggestion
    "ultrakill",
```